go/cli/mcap/README.md

```diff
@@ -162,3 +160,18 @@ The `mcap list` command can be used with chunks or attachments:
 To ensure the resulting binary is statically linked, build with `make`:
 
     make build
+
+### API Evolution Policy
+
+This tool uses [Semantic Versioning 2.0.0](https://semver.org/spec/v2.0.0.html) to indicate the scope of an update to users. In addition, we try to follow these guidelines when updating the functionality of the tool.
```

Comment:
if we are using semver, I'm not sure why we need the "one month" policy. They can always stay on the old version as long as they want, and depending on where in the month they encounter the feature/bug that causes them to want to upgrade the fact that it's been deprecated a month may not matter or even be known to them. People probably won't upgrade the tool until they are affected by a bug or want a new feature.

I wonder if we could replace the "one month" concept with a statement like, "bugs will be backpatched up to one release back" (if we are comfortable committing to that).

The hypothetical issue I see with this 1 month policy is,
1. User is using v1 with feature X
2. We release v2 that deprecates feature X. User doesn't care/isn't aware; v1 works fine.
3. +31 days We release v3 that introduces valuable feature Y and retires X. User didn't know that was coming, but now needs to upgrade.

In this scenario it's not clear the 30 day wait time bought us anything. User is free to stay on the old version and upgrade on their own schedule either way.

The alternate issue where it's a bug not a feature is,

1. User is using v1 with feature X
2. We release v2 that deprecates feature X. User doesn't care/isn't aware; v1 works fine.
3. +29 days User encounters a bug in v1/v2
4. +31 days We release v3 that introduces valuable feature Y fixes the bug, but drops X. User now needs to upgrade to get the bugfix, but also loses X.

In this case I don't think the month waiting period wasn't helpful, but what would be helpful would be a policy of backpatching bugs so the user doesn't need to take a breaking change to get the issue fixed.